tests/unit/models/reminder-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('reminder', 'Unit | Model | reminder', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
+
+test('it exists', function(assert) {
+  let model = this.subject();
+  // let store = this.store();
+  assert.ok(!!model);
```

Comment:
So this is a good start, but considering this is what it gives you I expect more. So I test that the model has a title, note, date, etc..